client/src/document/index.scss

```diff
@@ -96,6 +96,19 @@ a.new {
         background-color: transparent;
       }
     }
+
+    &[id^="attr-"] {
+      &:link,
+      &:visited {
+        color: $neutral-100;
+        text-decoration: none;
+      }
+
+      &:hover,
+      &:focus {
+        text-decoration: underline;
+      }
+    }
```

Comment:
Hey @KartikSoneji, we can remove this entire piece of `scss`. The styling works well without it, and it also avoids the problem this introduces with the `active` state. See the video below for how this will look and behave without the above.

https://user-images.githubusercontent.com/10350960/118502799-535c4f00-b72a-11eb-9f26-def614af8532.mov

Reply:
Can we leave the styling in?  
I feel that blue highlights the links too much, and we already discussed that underline on hover was more consistent with the styling of the `h2` elements.

Reply:
So, we would need to add more CSS to address the problem that currently exists with the active state. So, one of the other reasons I would rather we stick with this is that right now, these are not links and have never been so, users have no reason to try and use them as such.

If we stick with this styling then, visually nothing changes and users are bound to miss the fact that these are actually links now and might never discover this great new addition to the docs. Also, less code is generally better for maintenance and performance.

Reply:
Hey @ KartikSoneji, what if we do this. Keep the CSS you added but, remove the `color: $neutral-100;` line?

Reply:
Hi, apologies for the delay.

What about this?  
This keeps the original color text, while fixing the `active` state with minimal CSS.

```diff
--- a/client/src/document/index.scss
+++ b/client/src/document/index.scss
@@ -108,6 +108,10 @@ a.new {
       &:focus {
         text-decoration: underline;
       }
+
+      &:active {
+        color: $text-color-inverted;
+      }
     }
   }
```

This is what it looks like:
![image](https://user-images.githubusercontent.com/25801283/119063244-3fe50880-b9f6-11eb-8d13-1e386bdf47d4.png)

`:active`
![image](https://user-images.githubusercontent.com/25801283/119063327-728f0100-b9f6-11eb-9d0b-ab1a0cb7946e.png)

Reply:
Let’s do it @KartikSoneji

Reply:
@schalkneethling Great! I pushed a commit, please take a look.